js/models/roomcollection.js

```diff
@@ -28,7 +28,9 @@
 
 	var RoomCollection = Backbone.Collection.extend({
 		model: OCA.SpreedMe.Models.Room,
-		comparator: 'name',
+		comparator: function(model) {
+			return -(model.get('lastPing'));
```

Comment:
AFAIK 'lastPing' is never set on room objects, or at least I could not find any code that would set it